config.json

```diff
@@ -11,10 +11,10 @@
     "analyzer": false
   },
   "files": {
-    "solution": ["%{snake_slug}.f90"],
+    "solution": [".meta/example.f90"],
```

Comment:
I think you updated the wrong file here. You probably want to update the `example` key.

Reply:
Hmm, OK got confused but I reread: [https://exercism.lol/docs/building/tracks/config-json](https://exercism.lol/docs/building/tracks/config-json). I will fix accordingly.

So "solution" is the file that the exercism-user will use as template for creating their solution. I think work "template"  would be better in my mind, but OK. 
"example" is then the solution or the actual implementation of the exercise, which I thought was required but seems to be optional in doc. What is "examplar" and "editor"?